src/assets/styles/_media-dimensions.scss

```diff
@@ -1,6 +1,7 @@
-$min-bg-mob: 31.25rem; //500px
-$max-mob: 43.6875rem; //699px
-$min-tab: 43.75rem; //700px
-$max-tab: 79.9375rem; //1279px
-$min-desk: 80rem; //1280px
-$full-desk: 93.75rem; //1500px
+// grid-breakpoints from node_modules/bootstrap/scss/_variables.scss
+$breakpoint-xs: 0;
+$breakpoint-sm: 576px;
```

Comment:
mieliśmy w `rem`, teraz `px`, niżej margines w `em`
może wybierzmy jakąś jedną jednostkę?

Reply:
To jest akurat wytyczna bootstrap 5 więc nie widzę problemu dla tego przypadku: https://getbootstrap.com/docs/5.0/layout/breakpoints/

Reply:
w międzyczasie sam sparing w boostrapie jest na rem: https://getbootstrap.com/docs/4.0/utilities/spacing/

Reply:
px i rem współgrają, bardziej mi nie pasowały te em :)